storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -310,15 +314,16 @@ func (c *PrometheusConverter) addHistogramDataPoints(dataPoints pmetric.Histogra
 			infBucket.Value = float64(pt.Count())
 		}
 		infLabels := createLabels(baseName+bucketStr, baseLabels, leStr, pInfStr)
+		c.handleStartTime(startTimestamp, infBucket.Timestamp, infBucket.Value, infLabels, settings)
 		ts := c.addSample(infBucket, infLabels)
 
 		bucketBounds = append(bucketBounds, bucketBoundsData{ts: ts, bound: math.Inf(1)})
 		c.addExemplars(pt, bucketBounds)
 
-		startTimestamp := pt.StartTimestamp()
-		if settings.ExportCreatedMetric && startTimestamp != 0 {
+		otelStart := pt.StartTimestamp()
```

Comment:
Not in love with this name; everything around here is Otel.
AFAICS the difference is that this one is in nanoseconds while `startTimestamp` is in milliseconds; perhaps that could be used in the name?

Reply:
I can fix.

Reply:
I renamed `startTimestamp` to `startTimestampMs` and `otelStart` to `startTimestampNs`. PTAL.